content/java/concepts/math-methods/terms/negate-exact/negate-exact.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.negateExact()'
+Description: 'Returns the negation of the argument and raises an exception if the result overflows.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `Math.negateExact()` method returns the negation of the argument and raises an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows.
+
+## Syntax
+
+```pseudo
+Math.negateExact(num)
+```
+
+The `num` argument can be of type `int` or `long`.
```

Comment:
```suggestion
The `num` parameter can be of type `int` or `long`.
```

In the context of describing the makeup of a method/function, let's refer to any passed values as parameters and as arguments in examples. 😄